src/CitySearch.js

```diff
@@ -66,4 +68,9 @@ class CitySearch extends Component {
     }
 }
 
+CitySearch.propTypes = {
+    suggestion: PropTypes.string,
+    showSuggestions: PropTypes.string,
```

Comment:
These two keys already exist in the CitySearch Component as state variable. There is no need to include them there. Instead, you need to put all variables that you access from `this.props`. Like `locations`, `updateEvents`.

Reply:
```
CitySearch.propTypes = {
    locations: PropTypes.arrayOf(PropTypes.string).isRequired,
    updateEvents: PropTypes.func.isRequired,
}
```